CentralComputing/UnitTests/StateMachine_Test.cpp

```diff
@@ -1,53 +1,68 @@
-#include "Sensor.h"
-#include <iostream>
-#include <unistd.h>
-#include "Pod.h"
-#include "StateMachineCompact/stdafx.h"
-#include <string>
-#include <fstream>
-#include <sstream>
-#include <algorithm>
-#include <vector>
-#include <stdlib.h>
+#include "StateMachine_Test.h"
 
-#include <time.h>
+//Include any other dependencies 
 
-using namespace std;
-
-// declaring function, implented later
-void print_help();
-void print_help_instr();
+StateMachine_Test::StateMachine_Test(){
```

Comment:
Given Curtis and my plans to expand StateMachine_Test to be a full pod test suite integrating multiple components, I think it's probably best to not have it implement Unit_Test.  It doesn't really fit the idea of a unit test, it's more of an integration test and is kind of monolithic in its usage.  Thoughts?